### ISSUE TITLE
Avoid use of "comprise"

### ISSUE THREAD

**Issue by daveedvdv** [cplusplus/draft#2975]
The use of "comprise" has been increasing in the draft (e.g., compare N4791 to N4820). The uses are correct, but the word "comprise" is commonly misunderstood to be a synonym of "include" (it is not; it is a synonym of "consist of"), both by native English-language speakers and by people whose first language is not English.  I'd suggest using "consist of".

**Comment by jwakely** [comment]
Yes, every time I see it I have to look carefully to be sure it means what I think it means, because it's so commonly misused.

**Comment by jensmaurer** [comment]
We want to make a decision when @tkoeppe is present.

**Comment by jensmaurer** [comment]
Personal opinion: I think "comprise!" is exactly what we want to say here, popular misuses notwithstanding. We could consider adding an entry to our "terms an definitions" for this, to avoid any doubt about local use.

**Comment by zygoloid** [comment]
I'm sympathetic to the concerns about widespread misunderstanding of "comprise" (and I checked a couple of online dictionaries, which seem to list both the "correct" definition and the colloquial/"wrong" one); I generally prefer descriptive linguistics over prescriptive linguistics, and if popular usage is ambiguous, then we should look for a more-precise term. If our words confuse reasonable readers who are paying attention, that's a bug even if the words are technically correct.

That said, in formal, technical writing such as this, I think the expectations on usage lag significantly behind the vernacular (for example, we almost entirely eschew contractions), and the historically-correct interpretation should generally be understood as the intended one. When we say "literally", we don't mean "figuratively", and when we say "comprises", we mean "is composed exclusively of", not "form part or all of".

I don't think this is an easy choice. If we can find a good alternative that doesn't sacrifice clarity for the readers who are correctly interpreting the existing "comprise"s, I think that's worth pursuing, but I suspect any alternative here will be at least a little more verbose and/or less clear.

**Comment by daveedvdv** [comment]
Well, I think "consist of" is a synonym that works in all cases (modulo appropriate declensions) and addresses the concern.  Since there aren't too many uses of "comprise" yet, making the change at this time seems relatively painless to me.

**Comment by jensmaurer** [comment]
Editorial meeting: We found some misuses; fix those. Use "consists of" mostly, except where "comprises" cannot be misunderstood.
